pkgs/by-name/sd/SDL_ttf/package.nix

```diff
@@ -1,56 +1,57 @@
 {
   lib,
   SDL,
+  SDL_mixer,
   fetchpatch,
-  fetchurl,
+  fetchFromGitHub,
   freetype,
   stdenv,
+  unstableGitUpdater,
   # Boolean flags
   enableSdltest ? (!stdenv.hostPlatform.isDarwin),
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "SDL_ttf";
-  version = "2.0.11";
+  version = "2.0.11.1-unstable-2024-04-23";
 
-  src = fetchurl {
-    url = "https://www.libsdl.org/projects/SDL_ttf/release/SDL_ttf-${finalAttrs.version}.tar.gz";
-    hash = "sha256-ckzYlez02jGaPvFkiStyB4vZJjKl2BIREmHN4kjrzbc=";
+  src = fetchFromGitHub {
+    owner = "libsdl-org";
+    repo = "SDL_ttf";
+    rev = "3c4233732b94ce08d5f6a868e597af39e13f8b23";
+    hash = "sha256-FX6Ko4CaOSCSKdpWVsJhTZXlWk1cnjbfVfMDiGG2+TU=";
   };
 
-  patches = [
-    # Bug #830: TTF_RenderGlyph_Shaded is broken
-    (fetchpatch {
-      url = "https://bugzilla-attachments.libsdl.org/attachments/830/renderglyph_shaded.patch.txt";
-      hash = "sha256-TZzlZe7gCRA8wZDHQZsqESAOGbLpJzIoB0HD8L6z3zE=";
-    })
-  ];
-
-  patchFlags = [ "-p0" ];
-
   buildInputs = [
     SDL
     freetype
   ];
 
   # pass in correct *-config for cross builds
   env.SDL_CONFIG = lib.getExe' (lib.getDev SDL) "sdl-config";
-  env.FREETYPE_CONFIG = lib.getExe' freetype.dev "freetype-config";
+  env.FT2_CONFIG = lib.getExe' freetype.dev "freetype-config";
 
   configureFlags = [
     (lib.enableFeature enableSdltest "sdltest")
   ];
 
   strictDeps = true;
 
+  passthru.updateScript = unstableGitUpdater {
+    tagFormat = "release-2.0.11";
+    tagPrefix = "release-";
+    branch = "SDL-1.2";
+  };
+
   meta = {
     homepage = "https://github.com/libsdl-org/SDL_ttf";
     description = "SDL TrueType library";
     license = lib.licenses.zlib;
     maintainers = lib.teams.sdl.members ++ (with lib.maintainers; [ ]);
     inherit (SDL.meta) platforms;
     knownVulnerabilities = [
-      "CVE-2022-27470"
+      # CVE applies to SDL2 https://github.com/NixOS/nixpkgs/pull/274836#issuecomment-2708627901
```

Comment:
just drop it completely - the CVE had ambiguous wording ("releases prior to 2.0.18" or something like that) and i assumed that applied to 1.x versions. But with this debunked, there is no reason to keep any of this.

Reply:
the comment is there to avoid the CVE being added again by some unknowing contributor